core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/scan/text/FileAttributeType.java

```diff
@@ -0,0 +1,42 @@
+package edu.uci.ics.texera.workflow.operators.source.scan.text;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import edu.uci.ics.texera.workflow.common.tuple.schema.AttributeType;
+
+public enum FileAttributeType {
+    STRING("string", AttributeType.STRING),
+    SINGLE_STRING("single string", AttributeType.STRING),
+    INTEGER("integer", AttributeType.INTEGER),
+    LONG("long", AttributeType.LONG),
+    DOUBLE("double", AttributeType.DOUBLE),
+    BOOLEAN("boolean", AttributeType.BOOLEAN),
+    TIMESTAMP("timestamp", AttributeType.TIMESTAMP),
+    BINARY("binary", AttributeType.BINARY);
```

Comment:
please avoid wrapping an enum around another enum. This would introduce many confusion and dead code. I know the original `TextFileScanOp` is doing it, but we should take the chance to fix this problem. The only "diff" is `SINGLE_STRING`, so maybe consider making it a special case of `AttributeType.STRING` and provide a check box for the user to differentiate single string or not?

Reply:
@Yicong-Huang will find a solution for this.

Reply:
After inspection, we might be able to use scala enum to do something different. Let's fix it in a future PR.